packages/lit-html/src/lit-html.ts

```diff
@@ -1046,13 +1053,22 @@ class ChildPartImpl {
     this._$committedValue = value;
   }
 
-  private _commitTemplateResult(result: TemplateResult): void {
-    const {values, strings} = result;
-    const template = this._$getTemplate(strings, result);
+  private _commitTemplateResult(
+    result: TemplateResult | CompiledTemplateResult
+  ): void {
+    const {values} = result;
+    let template: Template | undefined;
+    if (typeof result._$litType$ === 'number') {
```

Comment:
This isn't a "type" anymore; maybe just `_$lit$`?

Reply:
It's kind of back to a type with it not being a function. The "type" is the template it references.